### DISCUSSION TITLE
How to secure source code or obfusicate or export as .sh file?

### DISCUSSION THREAD

**Discussion by CompoTeam** [swoole/swoole-src#4593] [Q&A]
Hi 

Is there any tool to encrypt a swoole source code or export it as a .sh file?

Thanks in advance.

**Comment by leocavalcante** [comment]
You can try https://business.swoole.com/compiler.html

**Reply by CompoTeam** [reply]
> You can try https://business.swoole.com/compiler.html

is this enterprise? and how does it works?

**Reply by CompoTeam** [reply]
any free tool or personal usage free plan?
